frontend/storage/common.ts

```diff
@@ -1,12 +1,23 @@
+// eslint-disable-next-line no-unused-vars
+import {GameSpoilers} from './models/GameSpoilers';
 export const SET_BREADCRUMB = 'SET_BREADCRUMB';
+export const SET_GAME_SPOILERS = 'SET_GAME_SPOILERS';
```

Comment:
I am trying to think of a better name here and I am drawing a blank. I am not fond of the "negative" being in the name (spoiler = invisible, it's generally easier to reason with fewer negative, eg. `isNotNull = true` is worse than `isNullable = false`). Maybe go with *allowed* instead?

And if that'll be just setting a single boolean you may make it `SET_IS_GAME_ALLOWED` instead.